src/olympia/reviewers/models.py

```diff
@@ -225,7 +225,10 @@ def base_query(self):
         query['where'].append(
             f'((addons.addontype_id IN ({types}) '
             'AND files.is_webextension = 0) '
-            'OR addons_addonreviewerflags.auto_approval_disabled = 1)')
+            'OR addons_addonreviewerflags.auto_approval_disabled = 1 '
+            'OR addons_addonreviewerflags.auto_approval_disabled_until > NOW()'
```

Comment:
Yes. The flag isn't necessarily cleared, it could stay at an old date, so if the date it's set to is before `NOW` then we don't need to take it into account.